### ISSUE TITLE
Commit listing results back

### ISSUE THREAD

**Issue by BastianZim** [apple/password-manager-resources#408]
Hi,

just wanted to raise this issue to see if this could maybe be implemented, as I don't know the necessary programming languages to create a PR myself :)

Currently, from what I could find, the linter raises an error if there is something wrong with the JSON (Not alphabetical etc.) in a PR and stops there.

I was wondering if you would consider implementing committing the correctly formatted JSON back to the PR? This should be doable, if the linter is able to not only check for a correct format but also create it itself, with something like this in the GitHub actions file:

```yaml
git config --local user.email "action@github.com"
git config --local user.name "actions-user"
git add $filename.JSON
git commit -m "Update JSON" -a
git push "https://${GITHUB_ACTOR}:${{ secrets.GITHUB_TOKEN }}@github.com/${GITHUB_REPOSITORY}.git" HEAD:${GITHUB_REF}
```

**Comment by Cldfire** [comment]
Yeah, I'd love to see a simple CI setup that would help keep the files to-standard :slightly_smiling_face:. This is a great suggestion!

Any such setup would need to be put together by @rmondello; I know there's been some discussion about it before, but I'm not sure if any conclusion has been reached or not.

**Comment by BastianZim** [comment]
Just to add, in python this should be pretty easy, at least for the _websites-with-shared-credential-backends.json_ file. I haven't tested this but I'd assume this should suffice:

```python
import json


with open("websites-with-shared-credential-backends.json") as json_file:
    raw_json = json.load(json_file)
    # Read the JSON file. Would need to check path.


sorted_json = []
for i in raw_json:
    unique_json = list(set(i))
   # For each company, establish uniqueness
    sorted_json.append(sorted(unique_json))
    # Sort each company.

return_json = sorted(sorted_json, key=lambda e: e[0])
# Sort the list of domains by the first subdomain
```
But not sure how to do that in ruby.
